extensions/sql-bindings/src/common/constants.ts

```diff
@@ -87,3 +87,4 @@ export const userPasswordLater = localize('userPasswordLater', 'In order to user
 export const openFile = localize('openFile', "Open File");
 export const closeButton = localize('closeButton', "Close");
 export function addSqlBinding(functionName: string): string { return localize('addSqlBinding', 'Adding SQL Binding to function "{0}"...'), functionName; }
+export const connectionProgressTitle = localize('connectionProgressTitle', "Connecting to SQL Server Connection...");
```

Comment:
Maybe something like `Testing SQL Server connection...`

(connecting to a connection doesn't make sense. And connection usually shouldn't be capitalized)